README.md

```diff
@@ -1 +1,108 @@
-# postgres-setup
\ No newline at end of file
+# postgres-setup
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🎦 Video Description](#video)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Vet Clinic DataBase <a name="about-project"></a>
+
+In this Repo I use PostgreSQL to create a relational database for a vet clinic.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- PostgreSQL
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+  <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+## Key Features
+
+- Create a table using `CREATE TABLE table_name()`
+- Add entries using `INSERT INTO table_name VALUES()`
+- Write queries to get information from the table
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`cd [your-folder] git@github.com:yonas44/postgres-setup.git`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@yonas44](https://github.com/yonas44)
+- LinkedIn: [Yonas Tesfu](linkedin.com/in/yonas-tesfu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/yonas44/postgres-setup/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] This link is broken because you don't have a MIT.md file in your repository :fearful: Kindly add a MIT.md file using these [link](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [ ] After adding this template file, you will need to add your **name**, **current year**, and **app-type** in your MIT.md file to make your project licensed :+1:

Reply:
Okay thank you, I will make sure to add them